tests/environments/container/Containerfile

```diff
@@ -0,0 +1,51 @@
+FROM centos:8
+
+# Python options = [3.8, 3.9]
+ARG PYTHON_VERSION=3.8
+ARG GOLANG_VERSION=1.15.7
+
+ENV PATH=${PATH}:/usr/local/go/bin
+
+RUN set -x \
+    && echo 'fastestmirror=True' >> /etc/dnf/dnf.conf \
+    && dnf update -y \
+    # Receptor build tools
+    && dnf install -y \
+        git wget make iproute openssl findutils virtualenv \
+    # Install specific python version
+    && export PYTHON_PKG_NAME=python$(echo ${PYTHON_VERSION} | sed 's/\.//g') \
+    && dnf module install -y ${PYTHON_PKG_NAME} \
+    && dnf clean all \
+    && alternatives --set python /usr/bin/python${PYTHON_VERSION} \
+    # Install specific golang version
+    && cd /tmp/ \
+    && wget -q https://golang.org/dl/go${GOLANG_VERSION}.linux-amd64.tar.gz \
+    && rm -rf /usr/local/go \
+    && tar -C /usr/local -xzf go${GOLANG_VERSION}.linux-amd64.tar.gz \
+    && rm -f go${GOLANG_VERSION}.linux-amd64.tar.gz \
+    && echo 'export PATH=${PATH}:/usr/local/go/bin' >> /etc/bashrc
```

Comment:
I'll need help for Go installation on CentOS 8.

If I run the Receptor CLI tests using the golang installation method above, then I got this error message:

```shell
+ go test -v ./functional/cli/cli_test.go
# runtime/cgo
cgo: exec /missing-cc: fork/exec /missing-cc: no such file or directory
FAIL    command-line-arguments [build failed]
FAIL
```

How to solve:

If I install the golang package using `yum`, everything works fine, but I can't specify a golang version this way.

```shell
yum install -y golang
```

I need help to find a way to make the first method works fine or find some way to install any (maybe not any) golang versions via `yum`.

Reply:
Was decided to use the stable version from CentOS 8 repo.